modules/hitl/assets/default.css

```diff
@@ -243,7 +243,7 @@
 }
 
 .bph-chat-bubble {
-  max-width: 60%;
+  max-width: 50%;
```

Comment:
Had to change this value for the carousel to be properly rendered. It would otherwise overflow and cause the chat to have a horizontal scrollbar. 

Not sure if this issue comes from the component itself or simply that requires the module to adapt its CSS!